lib/meterClient.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {agent} = require('bedrock-https-agent');
+const {Ed25519Signature2020} = require('@digitalbazaar/ed25519-signature-2020');
```

Comment:
I don't like the tight integration here -- but it will have to be factored out later.